experiment_helpers/run_checkLPC.m

```diff
@@ -0,0 +1,93 @@
+function [expt, exptPre] = run_checkLPC(expt, exptPre)
+% Runs a short speech experiment using run_measureFormants_audapter, then
+% plots formant means using check_audapterLPC. Useful for determining LPC
+% order and vowel means for participants in most speech experiments. The
+% indended use is that this will happen near the beginning of a study,
+% before the main phase. 
+% 
+% The result of this function is saving a file `exptPre`. The variable 
+% exptPre can also be passed back as an output argument.
+%
+% This function assumes the input argument `expt` has the following fields:
+%   expt.dataPath
+%   expt.snum
+%   expt.gender
+
+% 2022-11 Chris Naber init.
+
+%% setup
+if nargin < 2
+    exptPre = [];
+end
+
+defaultParams.snum = expt.snum;
+defaultParams.gender = expt.gender;
+defaultParams.dataPath = fullfile(expt.dataPath, 'pre');
+defaultParams.words = {'bid' 'bat' 'bed'};
+defaultParams.conds = {'noShift'};
+defaultParams.trackingFileLoc = 'experiment_helpers'; % default single-syllable, one word Audapter OST file
+defaultParams.trackingFileName = 'measureFormants';   % default single-syllable, one word Audapter OST file
+defaultParams.audapterParams.fb = 3;                  % feedback mode. 3=participant hears speech and noise
+if isfield(expt, 'bTestMode') && expt.bTestMode
+    defaultParams.nblocks = 2;  % number of repetitions of each word
+else
+    defaultParams.nblocks = 10; % number of repetitions of each word
+end
+defaultParams.breakTrials = exptPre.ntrials; % no breaks
```

Comment:
Has this been tested without specifying exptPre as an input arg? It looks like that would produce an error here, as it would be blank.